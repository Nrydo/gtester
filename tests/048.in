8090922
638723307
532314594