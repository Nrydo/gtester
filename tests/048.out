8686006999307737